### PR TITLE
Feature/eclipse angus

### PR DESCRIPTION
**Pull request by liquidz**
* Bumps outdated libraries
  * `com.sun.mail/jakarta.mail` moves to `org.eclipse.angus/angus-mail`
    * Jakarta Mail implementation moves to Eclipse Angus
      * https://jakartaee.github.io/mail-api/
      * https://eclipse-ee4j.github.io/angus-mail/
* Adds docker-compose.yml for local testing.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: "3"
```
Comment (nsyee) [inline docker-compose.yml]: @liquidz `docker-compose.yml` is already in [dev-resources](https://github.com/toyokumo/toyokumo-commons/blob/main/dev-resources/docker-compose.yml),  which is used in Makefile.
Is it for a different purpose?

**Review by nsyee — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (liquidz) [reply docker-compose.yml]: @nsyee Thanks for your confirmation!
Oh, I've missed the file.
I removed this file.

**Review by nsyee — APPROVED** [review]
@liquidz thanks! LGTM 👍

**Comment by liquidz** [comment]
Thanks!

**Pull request merged**